apps/shipments/serializers.py

```diff
@@ -234,14 +234,49 @@ def validate_delivery_act(self, delivery_act):
 
 class PermissionLinkSerializer(serializers.ModelSerializer):
     shipment_id = serializers.CharField(max_length=36, required=False)
+    emails = serializers.ListField(
+        child=serializers.EmailField(),
+        min_length=0,
+        required=False
+    )
+
+    def __init__(self, *args, **kwargs):
+        super(PermissionLinkSerializer, self).__init__(*args, **kwargs)
+        self.subject = None
```

Comment:
`subject`/`link` don't have anything to do with a PermissionLink, they are related to the email content, so I think it makes sense to move the construction of these to the view.